adele-ticket-master-api/src/main/java/com/microservicesteam/adele/ticketmaster/events/TicketsWereAlreadyBooked.java

```diff
@@ -0,0 +1,18 @@
+package com.microservicesteam.adele.ticketmaster.events;
+
+import com.microservicesteam.adele.messaging.events.Event;
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface TicketsWereAlreadyBooked extends Event {
```

Comment:
I suggest to keep extending TicketEvent, positions will be useful for tracking the information in the logs and also the UI needs the information anyway.